nmk/service_auth/only_message/call_store.py

```
@@ -0,0 +1,98 @@
+        "caller_id": caller_id,
+        "caller": caller_username,
+        "caller_pic": caller_pic or "",
+        "callee_id": callee_id,
+        "call_type": call_type,
+        "offer": offer,
+        "status": "ringing",
+        "created_at": time.time(),
+    }
+    try:
+        _redis().set(f"{CALL_KEY_PREFIX}{call_id}", json.dumps(payload), ex=RING_TTL)
+    except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)